deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -189,8 +189,6 @@ spec:
           - name: DEVICE_DISCOVERY_STRATEGY
             value: {{ .Values.deviceDiscoveryStrategy }}
         {{- end }}
-          - name: DEVICE_PLUGIN_MODE
-            value: "{{ .Values.devicePlugin.mode }}"
```

Comment:
What is this even?

Reply:
This was the `DEVICE_DISCOVERY_STRATEGY` before we reworked that. It was never released, but was missed when we moved to `DEVICE_DISCOVERY_STRATEGY`. It is unused and the values entry doesn't exist. I can do this in a separate PR if that is cleaner.

Reply:
No, it's fine